README.md

```diff
@@ -18,15 +18,29 @@ LEAP-Accelerate includes:
 
 `mkdir build && cd build`
 
-`mdir linux && cd linux`
+`mkdir linux && cd linux`
+
+`mkdir Debug`
+
+`mkdir Release`
 
 `export CUDA_HOME=/usr/local/cuda`
 
 `export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:${CUDA_HOME}/lib64:${CUDA_HOME}/extras/CUPTI/lib64`
 
 `export PATH=$PATH:$CUDA_HOME/bin`
 
-`cmake ../../ -DCUDA_TOOLKIT_ROOT_DIR="/usr/local/cuda"`
+#### Debug
+
+`cd Debug`
+
+`cmake ../../../ -DCUDA_TOOLKIT_ROOT_DIR="/usr/local/cuda" -DCMAKE_CXX_FLAGS_DEBUG=-O1 -DCMAKE_BUILD_TYPE=Debug`
+
+#### Release
+
+`cd Release`
+
+`cmake ../../../ -DCUDA_TOOLKIT_ROOT_DIR="/usr/local/cuda" -DCMAKE_CXX_FLAGS_RELEASE=-O3 -DCMAKE_BUILD_TYPE=Release`
```

Comment:
Passing `-DCMAKE_CXX_FLAGS_RELEASE=-O3` should be unnecessary when doing `-DCMAKE_BUILD_TYPE=Release`

Reply:
updated